js/token-tree.js

```diff
@@ -1,5 +1,8 @@
+/* global isMailing, verify */
+// These variables are defined in civicrm/templates/CRM/Mailing/Form/InsertTokens.tpl
+
 (function ($, CiviCaseBase) {
-  $(document).on('crmLoad', function (eventObj) {
+  $(document).off('crmLoad').on('crmLoad', function (eventObj) {
```

Comment:
Using `off` in this way will stop all listeners of this event on the document. If we want to avoid breaking listeners from other extensions or areas we might need to do something like this:

```js
$(document).off('crmLoad', onCrmLoad).on('crmLoad', onCrmLoad);

// might need a better function name:
function onCrmLoad (eventObj) {

}

```

By passing the handler function it will only remove this handler from listening to `crmLoad`